packages/asc-core/src/components/SocialBar/SocialBar.ts

```diff
@@ -0,0 +1,8 @@
+import styled from '../../styled-components'
+import ButtonBar from '../ButtonBar'
+
+type Props = {}
+
+const SocialBar = styled(ButtonBar)<Props>``
```

Comment:
If you don't pass in styling, is it still useful to make ButtonBar a styled component?

Reply:
👍